### ISSUE TITLE
Enhancement Request > Add words to the user dictionary from the editor

### ISSUE THREAD

**Issue by ljsinclair** [bartosz-antosik/vscode-spellright#512]
Thanks for the extension, it works really well.

My enhancement request is borne of two issues:
1. I'm new to VScode, having migrated because Atom editor is being retired at the end of the year
2. I don't actually know where to find the user-defined dictionary (you've said "...contained in spellright.dict files, one located in user settings folder and the other in workspace settings folder..."

As I say, I'm currently using Atom editor, with a built-in spell checker which has:
* RH click to add user-defined words to the dictionary
* Words are found and can be added to the spell-check settings page

Basically it'd be great to be able to add words via rh-click rather than having to manually open a standalone file then add them.

**Comment by vitoexe** [comment]
Yo Man !

You can add the string you want to "a.k.a. ignored words" by:

1.  selecting it
2. open command prompt (Ctrl+Shift+P / ⇧⌘P)
3.  write: SpellRight: Add Selection to .......

**Comment by antichris** [comment]
To fix a word that is marked as misspelled:

1. move the text cursor to it
2. activate the *Quick Fix* pop-up menu (<kbd>Ctrl</kbd>+<kbd>.</kbd> / ⌘. or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> / ⇧⌘P and type "quick fix")
3. select the action you wish to perform (correct to a suggestion or add to one of the dictionaries) from the menu.

**Comment by ljsinclair** [comment]
Thanks for the feedback. 
I've decided to abandon VScode for various reasons.
Hope the plugin goes well :)
